PoS space review (#501)
0xPolygon/polygon-docs: docs/pos/how-to/full-node/full-node-ansible.md

comments: true
---

An [Ansible playbook](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html) is used to configure and manage a full node. 

## Prerequisites

> - Install Ansible on your local machine with Python3.x. The setup will not work if you have Python 2.x.

Reviewer: ```suggestion
- Install Ansible on your local machine with Python3.x. The setup doesn't run on Python 2.x.
```

    - To install Ansible with Python 3.x, you can use pip. If you do not have pip on your machine,
      follow the steps outlined [here](https://pip.pypa.io/en/stable/). Run `pip3 install ansible` to install
      Ansible.
- Check the [Polygon PoS Ansible repository](https://github.com/maticnetwork/node-ansible#requirements) for
  requirements.

> - You will also need to ensure that Go is *not installed* in your environment. You will run into issues if you attempt to set up your full node through Ansible with Go installed as Ansible requires specific packages of Go to be installed.

Reviewer: ```suggestion
- You also need to ensure that Go is *not installed* in your environment. You will run into issues if you attempt to set up your full node through Ansible with Go installed as Ansible requires specific packages of Go.
```
